### PR TITLE
small fixes, adding spatial search attributes latitude and longitude

### PR DESCRIPTION
**Pull request by sschlick**
here some small fixes as well as logitude/latitude search attributes

### CODE REVIEW

File: PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl [hunk 1]
```diff
@@ -24,7 +24,7 @@
   <xsl:variable name="implement-interfaces-firstpart">
     <xsl:choose>
       <xsl:when test="$generate-entity/parameters/parameter[@name='generate-enumeration']='true'">
-        <xsl:text>de.provision.framework.jpa.IdEnum</xsl:text>
+        <xsl:text>de.vivento.koop.util.IdEnum</xsl:text>
```
Comment (stefanseifert) [inline PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl]: please do not reference project-specific code here.

File: PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl [hunk 2]
```diff
@@ -93,7 +93,7 @@ package </xsl:text><xsl:value-of select="$platform/parameters/parameter[@name='e
       <xsl:text>@org.hibernate.annotations.TypeDefs({
   @org.hibernate.annotations.TypeDef(
     name="</xsl:text><xsl:value-of select="$entity"/><xsl:text>$</xsl:text><xsl:value-of select="@name"/><xsl:text>",
-    typeClass=de.provision.framework.jpa.XmlMappingUserType.class,
+    typeClass=de.vivento.koop.jpa.XmlMappingUserType.class,
```
Comment (stefanseifert) [inline PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl]: please do not reference project-specific code here.

File: PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl [hunk 3]
```diff
@@ -1188,6 +1219,18 @@ package </xsl:text><xsl:value-of select="$platform/parameters/parameter[@name='e
     }
   }
 
+  /**
+   * Konverter für diese Werteliste
+   */
+  public static class Converter extends de.vivento.koop.jpa.converter.WLConverter&lt;</xsl:text><xsl:value-of select="$entity"/><xsl:text>&gt; {
```
Comment (stefanseifert) [inline PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl]: please do not reference project-specific code here.

**Review by stefanseifert — CHANGES_REQUESTED** [review]

File: PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl (hunk 1 above)
Comment (sschlick) [reply PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl]: ok, I removed project-specific references with commit ebc8995

File: PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl (hunk 2 above)
Comment (sschlick) [reply PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl]: ok, I removed project-specific references with commit ebc8995

File: PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl (hunk 3 above)
Comment (sschlick) [reply PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl]: ok, I removed project-specific references with commit ebc8995

File: PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl [hunk 4]
```diff
@@ -1188,6 +1219,26 @@ package </xsl:text><xsl:value-of select="$platform/parameters/parameter[@name='e
     }
   }
 
+  /**
+   * Konverter für diese Werteliste
+   */
+  public static class Converter implements javax.persistence.AttributeConverter&lt;</xsl:text><xsl:value-of select="$entity"/>, Integer<xsl:text>&gt; {
```
Comment (stefanseifert) [inline PVEntityGenerator/xml/Platform_Java11JPA/entity_entity-class.xsl]: please use english language for comment

does this inner class need to be public? if possible, use default scope instead to not "pollute" the list of public classes in the project.

File: PVEntityGenerator/xml/Platform_Java11JPA/platform.xml [hunk 5]
```diff
@@ -271,6 +271,15 @@
           description="Mark this entity to be fulltext-indexed by Hibernate Search."
         ></parameter-definition>
 
+		  <parameter-definition
```
Comment (stefanseifert) [inline PVEntityGenerator/xml/Platform_Java11JPA/platform.xml]: this is a duplicate of the entry before?

**Review by stefanseifert — APPROVED** [review]
thanks!

**Pull request merged**